frontend/src/components/Output/ModelParameters.js

```diff
@@ -13,19 +10,28 @@ class ModelParameters extends Component {
             <table className="table table-bordered table-sm">
                 <thead>
                     <tr className="table-primary">
-                        <th colSpan="2">Model Parameters</th>
+                        <th colSpan="3">Model Parameters</th>
                     </tr>
                     <tr>
                         <th>Variable</th>
                         <th>Parameter</th>
+                        <th>Bounded</th>
                     </tr>
                 </thead>
                 <tbody>
-                    {_.map(store.selectedParams, (value, key) => {
+                    {store.modelParameters.map((params, idx) => {
+                        var [variable, param, bounded] = params;
                         return (
-                            <tr key={key}>
-                                <td>{key}</td>
-                                <td>{ff(value)}</td>
+                            <tr key={idx}>
+                                <td>{variable}</td>
+                                <td>{ff(param)}</td>
+                                <td>
+                                    {bounded.toString() === "true" ? (
```

Comment:
use `src/common/readOnlyCheckbox` instead of coding a new method - why convert a bool to a string for this comparison? just use the bool

Reply:
using readOnlyCheckbox .